eox_nelp/pearson_vue/pipeline.py

```diff
@@ -100,6 +100,8 @@ def get_user_data(user_id, **kwargs):  # pylint: disable=unused-argument
     pn = phonenumbers.parse(phone) if phone.startswith("+") else phonenumbers.parse(phone, profile.country.code)
     phone = str(pn.national_number)
     phone_country_code = str(pn.country_code)
+    extrainfo = getattr(user, "extrainfo", None)
+    arabic_name = extrainfo.arabic_name if extrainfo else ""
```

Comment:
Some cases the field has two words could we split that into first_arabic_name and last_arabic_name? I'm not sure if they uses middle name but if that is not the case probably if there are two words  that should be  first and last name...?

Reply:
But also could have 4 words, is not easier to split them or wich one is firstName or Last ??
https://lms.futurex.sa/admin/custom_reg_form/extrainfo/94813/change/